### ISSUE TITLE
Revisions tab: Focus at incorrect location after receiving errors

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#87]
URL: https://hcdev.openplus.ca/en/node/2525/revisions
User Agent: Chrome, Firefox
Language: Both

Issue Description: When user clicks on Compare selected revisions and there is an error on the page, the focus goes to the top of the page. User using keyboard has to tab through all the top elements on the page to get to the error.

Steps to reproduce: Go to Revisions tab and don't select anything. Hit enter on Compare selected revisions button.

Expected results: When there are errors on the page, the focus should be at the error messages on top so the user does not have to tab through all top elements. The screen reader user will also miss the error and has to find it.

![image](https://user-images.githubusercontent.com/14280062/129397095-c9824ead-0102-4bd9-ab25-14b7b03d8824.png)

**Comment by davidgotlieb** [comment]
OpenPlus has reported the issue to the owner of the module.

**Comment by mmiron-HC** [comment]
@davidgotlieb  This has not been resolved. The issues still persists. if this is a Drupal limitation, please tag it as such but this needs to stay as not fixed.
